backend/src/main/kotlin/org/loculus/backend/controller/SubmissionController.kt

```diff
@@ -252,23 +242,23 @@ class SubmissionController(
     fun getReleasedData(
         @PathVariable @Valid organism: Organism,
         @RequestParam compression: CompressionFormat?,
+        @RequestHeader(value = HttpHeaders.IF_NONE_MATCH, required = false) ifNoneMatch: String?,
     ): ResponseEntity<StreamingResponseBody> {
+        val lastDatabaseWriteETag = releasedDataModel.getLastDatabaseWriteETag()
+        if (ifNoneMatch == lastDatabaseWriteETag) return ResponseEntity.status(HttpStatus.NOT_MODIFIED).build()
```

Comment:
It would also be good to add the HttpStatus.NOT_MODIFIED response possibility to Swagger.

Reply:
We've done this now, please rereview